src/app/components-small/plugin-filter-editor/plugin-filter-editor.component.html

```diff
@@ -1,8 +1,14 @@
 <div class="filter-editor mat-elevation-z0">
     <h3 class="editor-description">Filter Editor</h3>
-    <mat-slide-toggle [(ngModel)]="showEditor">
-        Editor Mode {{ showEditor ? '(UI)' : '(JSON)' }}
-    </mat-slide-toggle>
+    <div class="editor-header">
+        <mat-slide-toggle [(ngModel)]="showEditor">
+            Editor Mode {{ showEditor ? '(UI)' : '(JSON)' }}
+        </mat-slide-toggle>
+        <button mat-raised-button type="button" (click)="ngOnInit()">
+            <mat-icon>refresh</mat-icon>
```

Comment:
This is a decorative icon, so it should be hidden from screen readers: <https://material.angular.io/components/icon/overview#decorative-icons>